src/Microsoft.DotNet.Cli.Utils/DangerousFileDetector.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.PlatformAbstractions;
+using RuntimeEnvironment = Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal class DangerousFileDetector : IDangerousFileDetector
+    {
+        public bool IsDangerous(string filePath)
+        {
+            if (RuntimeEnvironment.OperatingSystemPlatform != Platform.Windows)
+            {
+                return false;
+            }
+
+            return InternetSecurity.IsDangerous(filePath);
+        }
+
+        internal class InternetSecurity
```

Comment:
Can this be private? It can also be static.

Reply:
Fix, mark private and static